_docs/03.2-build_server_plugin.markdown

```diff
@@ -15,97 +15,97 @@ We are going to use the [GitHub Api](https://developer.github.com/v3/) to grab t
 The GitHub [site](https://developer.github.com/v3/) has great documentation and awesome libraries to help jumpstart our plugin. Let's begin there by installing a Node.js wrapper called [node-github](https://github.com/mikedeboer/node-github) in Your Awesome App:
 
 ```bash
-npm install github
+$ npm install github
 ```
 
 At their core, plugins are a simple `register object`, that has the `signature function(server, options, next)`. Read more about building plugins from scratch in the [Hapi documentation](http://hapijs.com/tutorials/plugins).
 
-Navigate to the `<your-awesome-app>/server/plugins` folder. Make a folder called `friends`. Cd into your new `friends` folder and:
+Navigate to the `<your-awesome-app>/server/plugins` folder. Make a folder called `friends`, `cd` into your new `friends` folder, and create an empty `index.js` file:
 
 ```bash
-  touch index.js
+$ touch index.js
 ```
 Your Server file pattern should now look like this:
 
 {% raw  %}
 ```
-  server
-    |_plugins
-      |_friends
-        |_index.js
-      |_webapp
-        |_index.html
+server
+  |_plugins
+    |_friends
       |_index.js
-    |_views
-      |_index-view.jsx
+    |_webapp
+      |_index.html
     |_index.js
```

Comment:
@caoyangs is there not also one under `server` ([looking at this](https://github.com/electrode-io/electrode-boilerplate-universal-react-node/blob/master/server/index.js)). The full tree snippet:

```
server
    ├── index.js
    ├── plugins
    │   └── webapp
    │       ├── index.html
    │       └── index.js
    └── views
        └── index-view.jsx
```